identity/identity.go

```diff
@@ -52,6 +68,14 @@ type (
 		// required: true
 		SchemaURL string `json:"schema_url" faker:"-" db:"-"`
 
+		// State is the identity's state.
+		//
+		// required: true
```

Comment:
I believe that for Enum you can use:

```suggestion
		// enum:
		// - active
		// - inactive
		// required: true
```